components/brave_wallet/common/brave_wallet.mojom

```diff
@@ -883,7 +888,7 @@ interface JsonRpcService {
   GetERC1155Metadata(string contract, string token_id, string chain_id) => (string response, ProviderError error, string error_message);
 
   // ENS lookups
-  EnsGetEthAddr(string domain) => (string address, ProviderError error, string error_message);
+  EnsGetEthAddr(string domain, EnsOffchainLookupOptions? options) => (string address, bool require_offchain_consent, ProviderError error, string error_message);
```

Comment:
Was unsure here if we are allowed to extend ProviderError instead of a separate flag. 
@yrliou wdyt?